File: core/Core/Models/UserCredential.cs
using System;
using System.Collections.Generic;

namespace Core.Models;

public partial class UserCredential
{
    public int CustomerId { get; set; }

    public string Username { get; set; } = null!;

    public string PasswordHash { get; set; } = null!;

    public DateTime? UpdatedAt { get; set; }

    public virtual Customer Customer { get; set; } = null!;
}

File: analytic/domain/user_credential/user_credential.py
from datetime import datetime


class UserCredential:
    def __init__(self,
                 customer_id: int,
                 username: str,
                 password_hash: str,
                 updated_at: datetime | None):
        self.customer_id = customer_id
        self.username = username
        self.password_hash = password_hash
        self.updated_at = updated_at
